README.md

```diff
@@ -1,15 +1,6 @@
 # starttls-policy-cli Python package
 
-## Package API
-
-## Run CLI
-Since this is a developer alpha, we recommend using `virtualenv` and `pip` to
-install and run `starttls-policy-cli`. To get set up:
-```
-virtualenv --no-site-packages --setuptools venv --python python2.7
-source ./venv/bin/activate
-pip install starttls-policy-cli
-```
+Run `pip install starttls_policy_cli` to install!
```

Comment:
Why or how did this change from hyphens to underscores?

Reply:
Sorry for interference, but recently I had this pain too. According to python syntax and naming conventions (like PEP-8) only valid word separator for package names in underscore. `setup.py` in this project has correct name `starttls_policy_cli`.

Python package index (PyPI) biases towards hyphen-separated package names. Even if you upload package with name separated with underscores, it will display package name with hyphens everywhere. But both invokations `pip install package_name` and `pip install package-name` are correct.

Reply:
But I'd prefer `pip install starttls-policy-cli` since pypi directly suggests it: https://pypi.org/project/starttls-policy-cli/

Reply:
No problem! What @Snawoot said is correct. Package naming conventions tend towards underscore, but PyPI biases towards hyphens. It's generally pretty confusing and I made a typo here.